azure/compute/src/main/resources/META-INF/services/org.jclouds.providers.ProviderMetadata

```diff
@@ -0,0 +1 @@
+org.jclouds.azurecompute.AzureComputeProviderMetadata
```

Comment:
We use Google Auto for service loader discovery. Remove this file and annotate the `AzureComputeProviderMetadata` accordingly.